<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>controlwidgetdata</class>
 <widget class="QWidget" name="controlwidgetdata">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label_6">
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>log</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="logSlider">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Left&lt;/span&gt;   = data not received, no widget visible&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Mid&lt;/span&gt;    = data received, but no widget visible&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Right&lt;/span&gt; = data received, widget visible according to saved config&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDial" name="dial">
     <property name="toolTip">
      <string>Left = data not received, no widget visible
Mid = data received, but no widget visible
Right = data received, widget visible according to saved config</string>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="sliderPosition">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>plot</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="plotSlider">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Left&lt;/span&gt;   = data not received, no widget visible&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Mid&lt;/span&gt;    = data received, but no widget visible&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Right&lt;/span&gt; = data received, widget visible according to saved config&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDial" name="dial_2">
     <property name="toolTip">
      <string>Left = data not received, no widget visible
Mid = data received, but no widget visible
Right = data received, widget visible according to saved config</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>26</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>gantt</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="ganttSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDial" name="dial_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Left&lt;/span&gt; = data not received, no widget visible&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Mid&lt;/span&gt; = data received, but no widget visible&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Right&lt;/span&gt; = data received, widget visible according to saved config&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
